pages/common/aws-sts.md

```diff
@@ -0,0 +1,12 @@
+# aws-sts
+
+> Security Token Service (STS) enables you to request temporary credentials for (IAM) users or for users that you authenticate (federated users).
```

Comment:
We try to avoid `you`/`your`
```suggestion
> Security Token Service (STS) allows to request temporary credentials for (IAM) users or for federated users.
```